core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -253,6 +253,18 @@ export class RteHelperModalComponent {
         );
         return;
       }
+    } else if (this.componentId === this.COMPONENT_ID_TABS) {
+      // Value[0] corresponds to tab Contents.
```

Comment:
What is "tab Contents"? There are multiple tabs, so not sure what you mean -- is it the contents for all tabs or just the first one? Perhaps worth clarifying your comment here.

Reply:
I altered the comment, yes, it is the contents for all tabs. I believe it makes more sense now. Thank you !